action.yml

```
@@ -20,6 +20,11 @@ inputs:
       used to pass the Dockerfile as $TIMESTAMP_TAG
     required: false
     default: ''
+  image_name:
+    description: 'Parameter to specify image container'
```

```suggestion
    description: 'Name of the image to build, e.g. "opensciencegrid/software-base"'
```

Examples are generally nice in help output